dallinger/experiment.py

```diff
@@ -1707,6 +1734,7 @@ def is_experiment_class(cls):
     )
 
 
+@cache
```

Comment:
I think you should remove this, it is what is causing the tests to fail, and the original motivation for caching it is no longer there